framework/PostProcessors/Metric.py

```diff
@@ -152,24 +169,34 @@ def inputToInternal(self, currentInputs):
       elif inputType == 'HDF5':
         self.raiseAnError(IOError, "Input type '", inputType, "' can not be accepted")
       elif inputType == 'PointSet':
-        pass #Allowed type
+        hasPointSet = True
```

Comment:
one variable is enough.
```hasDataObject```

```hasDataObject = True```

Reply:
I need both, since both PointSet and HistorySet can be inputted. The data will be extracted from both PointSet and HistorySet.